draft-ietf-rats-eat.md

```diff
@@ -405,14 +348,13 @@ It is also possible that some claims in the Evidence will be forwarded unmodifie
 This forwarding is subject to the verifier's implementation and policy.
 The relying party should be aware of the verifier's policy to know what checks it has performed on claims it forwards.
 
-The verifier may also modify or transform claims it forwards.
-This may be to implement some privacy preservation functionality.
+The verifier may modify claims it forwards, for example, to implement a privacy preservation functionality.
+The verifier MUST provide authenticity and integrity protection for attestation results.
```

Comment:
I would like to leave out part about providing authenticity and integrity here. It's not really part of the flow of what is being discussed and this section isn't really normative text.